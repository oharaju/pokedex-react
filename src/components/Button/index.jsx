function Button () {

  return (
    <div>
      <button>clique aqui</button>
    </div>
  )
};

export default Button;