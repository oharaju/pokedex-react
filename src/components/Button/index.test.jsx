import renderer from 'react-test-renderer';
import Button from './index';

it('button structure', () => {
  const tree = renderer.create(<Button/>).toJSON();
  expect(tree).toMatchSnapshot();
});

