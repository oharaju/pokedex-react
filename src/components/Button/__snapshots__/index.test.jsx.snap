// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button structure 1`] = `
<div>
  <button>
    clique aqui
  </button>
</div>
`;
